oviewer/input_search_test.go

```diff
@@ -278,14 +278,14 @@ func Test_stripBackSlash(t *testing.T) {
 		want string
 	}{
 		{
-			name: "nonEsacpe",
+			name: "nonEscaape",
```

Comment:
I cannot stop laughing, thanks @noborus 

Try this


```suggestion
			name: "nonEscape",
```

Reply:
...sorry